packages/cli-kit/src/public/node/conf.ts

```diff
@@ -1,3 +1,30 @@
-import Conf, {Schema} from 'conf'
+import Config from 'conf'
 
-export {Conf, Schema}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export class Conf<T extends {[key: string]: any}> {
```

Comment:
`LocalStorage`? I think it's more representative than `config` because we are starting to use it for caching purposes too, which is not configuration.